### PR TITLE
Changed routes for V1 and V2

### PR DESCRIPTION
**Pull request by JHUShanZhong**
V1 stuff => version: "v1", route "v1"
V2 stuff => version: "v2", route "v1"
fixed tests in helper/load_test.go

### CODE REVIEW

**Comment by JHUShanZhong** [comment]
[RV1: Change SYNQ-Golang SDK to use /v1/](https://trello.com/c/P30l2P8Y/211-rv1-change-synq-golang-sdk-to-use-v1)

File: test_server/server.go [hunk 1]
```diff
@@ -20,27 +20,29 @@ var recvParams []upload.UploadParameters
...
+	PROJECT_ID          = "1abfe1b849154082993f2fce78a16fda"
+	PROJECT_ID2         = "963bab6186a352b6c0e9de5d29418be3"
+	LIVE_VIDEO_ID       = "ec37c42b4aab46f18003b33c66e5e641"
+	API_KEY             = "aba179c14ab349e0bb0d12b7eca5fa24"
+	API_KEY2            = "cba179c14ab349e0bb0d12b7eca5fa25"
+	UPLOAD_KEY          = "projects/0a/bf/0abfe1b849154082993f2fce77a16fd9/uploads/videos/55/d4/55d4062f99454c9fb21e5186a09c2115.mp4"
+	INVALID_UUID        = `{"url": "http://docs.synq.fm/api/v1/errors/invalid_uuid","name": "invalid_uuid","message": "Invalid uuid. Example: '1c0e3ea4529011e6991554a050defa20'."}`
+	VIDEO_NOT_FOUND     = `{"url": "http://docs.synq.fm/api/v1/errors/not_found_video","name": "not_found_video","message": "Video not found."}`
+	API_KEY_NOT_FOUND   = `{"url": "http://docs.synq.fm/api/v1/errors/not_found_api_key","name": "not_found_api_key","message": "API key not found."}`
+	HTTP_NOT_FOUND      = `{"url": "http://docs.synq.fm/api/v1/errors/http_not_found","name": "http_not_found","message": "Not found."}`
```
Comment (jessica-christie) [inline test_server/server.go]: are these still valid for v1 or should they be changed to v0?

File: test_server/server.go [hunk 2]
```diff
@@ -192,6 +194,8 @@ func (s *TestServer) handle(w http.ResponseWriter, r *http.Request) {
 	case "v2":
 		s.handleV2(w, r)
```
Comment (jessica-christie) [inline test_server/server.go]: should this be removed?

**Comment by jessica-christie** [comment]
@JHUShanZhong @batmany13 what is the plan for the functions that have v1/v2 in their name?

File: helper/load_test.go [hunk 3]
```diff
@@ -30,15 +30,15 @@ func setup() (synq.ApiV2, Cache) {
...
+	server := test_server.SetupServer(SYNQ_VERSION, sampleDir)
 	url := server.GetUrl()
 	api.SetUrl(url)
 	api.UploadUrl = url
 	return api, cache
 }
 
 func (c Cache) GetCacheFile(name string) string {
-	return c.Dir + "/" + name + ".json"
+	return "./" + c.Dir + "/" + name + ".json"
```
Comment (batmany13) [inline helper/load_test.go]: c.Dir could be a "full path", so putting a "./" in front would be wrong.  Any particular reason why this is set?

Comment (JHUShanZhong) [reply helper/load_test.go]: When I do testing, the path without "./" would return false in os.Stat(). I don't know exactly why.

Comment (JHUShanZhong) [reply helper/load_test.go]: The original code works properly now. Might be some other reasons caused the error in previous test. I just changed it back.

**Comment by batmany13** [comment]
@jessica-christie we have to keep the v1 for now since there still could be services that use it (aerico, pixie etc).  Once we completely shutdown obaku, we can rewrite this.

**Pull request merged**